instructure/canvas-ios
# Add editing recipients from Compose

refs: MBL-13765
affects: parent
release note: none

Test plan:
* Enable Inbox feature flag
* Inbox > Compose message > CS 1400 > Icon in top right to edit recipients
* Add and remove recipients then tap the shadowy area to dismiss
* Recipient pills should update with the selected recipients
* ~I didn't add the empty state for the recipients field. I can, or we can do it separately?~
* Removing all recipients should display a To placeholder
